MagaluCloud/magalu
# feat(network): add connection resource blueprints

## Description

This PR adds the connection resources of Port-SecurityGroup attachment and PublicIP-Port attachment. It also removes, for now, the OpenAPI version of these resources

## Related Issues

- Closes #779

## How to test it

- Create an `mgc_network_port`, an `mgc_network_security_group` and an `mgc_network_public_ip` resource in TF
- Create the attachment resources
- See if they attach, detach correctly
